cohorts/variant_stats.py

```diff
@@ -158,6 +160,30 @@ def maf_somatic_variant_stats(variant, variant_metadata):
         normal_stats = _maf_variant_stats(variant, variant_metadata, prefix="n")
     return SomaticVariantStats(tumor_stats=tumor_stats, normal_stats=normal_stats)
 
+def _vcf_is_strelka(variant_file):
+    """Return True if variant_file given is in strelka format
+    """
+    return "strelka" in variant_file.lower()
+
+def _vcf_is_maf(variant_file):
+    """Retrun True if variant_file given is in .maf format
+    """
+    return ".maf" in variant_file.lower()
+
+def _vcf_is_mutect(variant_file):
+    """Return True if variant_file give is in mutect format
+    """
+    if "mutect" in variant_file.lower():
+        return True
+    else:
+        try:
+            vcf_reader = vcf.Reader(open(variant_file, "r"))
```

Comment:
Why do you need to open the VCF vs. use the `variant_metadata` already extracted by `varcode`?